web/api/v1/precheck/[app_id]/graphql/app-precheck.generated.ts

```diff
@@ -42,6 +42,9 @@ export type AppPrecheckQueryQuery = {
       app_flow_on_complete?: unknown | null;
       webhook_uri?: string | null;
       webhook_pem?: string | null;
+      return_to_ios?: string | null;
+      return_to_android?: string | null;
+      return_url?: string | null;
```

Comment:
@chipp What is return_url for?

Reply:
current version of iOS uses this key. for compatibility purposes let's have it here temporarily

Reply:
@chipp Could you elaborate? Is the current version of iOS using a key that doesn't exist? Is that why "Continue" button is non-responsive?